comptests/prov/test_prov_data_only_deploy.py

```diff
@@ -0,0 +1,149 @@
+                                    REPO=self.script_remote_branch)
+        LOGGER.info(cmd)
+        for node_obj in self.host_list:
+            node_obj.execute_cmd(cmd)
+        LOGGER.info("Step 1: Done.")
+
+        LOGGER.info("Step 2: Update solution.yaml with default data only deploy config.")
+        remote_sol_path = self.prov_deploy_cfg["git_remote_path"] + "solution.example.yaml"
+        solution_path = self.master_node_obj.copy_file_to_local(remote_path=remote_sol_path,
+                                                                local_path=self.local_sol_path)
+        assert_utils.assert_true(solution_path[0], solution_path[1])
+        resp = self.deploy_lc_obj.update_sol_for_granular_deploy(file_path=self.local_sol_path,
+                                                            host_list=self.host_list,
+                                            master_node_list=self.master_node_obj,
+                                            image=self.cortx_data_image,
+                                        deployment_type=self.deploy_cfg["deployment_type_data"])
+
+        assert_utils.assert_true(resp[0])
+        for node_obj in self.host_list:
+            res = node_obj.copy_file_to_remote(local_path=resp[1],
+                                               remote_path=self.prov_deploy_cfg[
+                                    "git_remote_path"] + self.deploy_cfg["solution_file"])
+            assert_utils.assert_true(res[0], res[1])
+        LOGGER.info("Step 2: Done.")
+
+        LOGGER.info("Step 3: Run prereq script on setup nodes.")
+        for node_obj in self.host_list:
+            self.deploy_lc_obj.execute_prereq_cortx(node_obj,
+                                                    self.prov_deploy_cfg["git_remote_path"],
+                                                    self.prov_deploy_cfg["pre_req"])
+        LOGGER.info("Step 3: Done.")
+
+        LOGGER.info("Step 4: Start data only cortx cluster.")
+        resp = self.deploy_lc_obj.deploy_cluster(self.master_node_obj,
+                                                 self.prov_deploy_cfg["git_remote_path"])
+        assert_utils.assert_true(resp[0], resp[1])
+        LOGGER.info("Step 4: Done.")
+        LOGGER.info("Test Completed.")
+
+    @pytest.mark.lc
+    @pytest.mark.comp_prov
+    @pytest.mark.tags("TEST-39370")
+    def test_39370(self):
+        """
+        Verify pods status for data only pod deployment.
+        """
+        LOGGER.info("Test Started.")
+        LOGGER.info("Check PODs are up and running.")
+        resp = self.deploy_lc_obj.check_pods_status(self.master_node_obj)
+        assert_utils.assert_true(resp)
+        LOGGER.info("All PODs are up and running.")
+        LOGGER.info("Test Completed.")
+
+    @pytest.mark.lc
+    @pytest.mark.comp_prov
+    @pytest.mark.tags("TEST-39333")
+    def test_39333(self):
+        """
+        Verify S3 status for all pods.
```

Comment:
are we verifying s3 status here or just ioservice status?

Reply:
s3 status